flocker/ca/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+A minimal certificate authority.
+"""
```

Comment:
Would anything be easier if this were part of the `flocker.cli` package?

Does that even make sense?  Which OS package will this be part of?  `clusterhq-flocker-cli`?

Would this be any better as `flocker ca` instead of `flocker-ca`?